hawk/app/views/resources/show.html.haml

```diff
@@ -10,121 +10,102 @@
 
-        - if @resource.running_on.any?
-          .panel.panel-default
-            .panel-heading#r-instances{role: :tab}
-              %h4.panel-title
-                %a{role: :button, data: {toggle: :collapse, parent: "#r-info"}, href: "#r-instances-body", aria: {expanded: false, controls: "r-instances-body"}}
-                  = _("Instances")
-                  .pull-right
-                    %span.caret
-            #r-instances-body.panel-collapse.collapse{role: :tabpanel, aria: {labelledby: "r-instances"}}
-              .panel-body
-                %table.table
-                  %tr
-                    %th= _("Node")
-                    %th= _("State")
-                  - @resource.running_on.each do |node, state|
-                    %tr
-                      %td= content_tag :code, node
-                      %td= content_tag :code, state || "-"
+      - if @resource.running_on.any?
+        .panel.panel-default
+          .panel-heading#r-instances{role: :tab}
+            %h4.panel-title
+              %a{role: :button, data: {toggle: :collapse}, href: "#r-instances-body", aria: {expanded: false, controls: "r-instances-body"}}
+                = _("Instances")
+                .pull-right
+                  %span.caret
+          #r-instances-body.panel-collapse.collapse{role: :tabpanel, aria: {labelledby: "r-instances"}}
+            %table.table
+              %tr
+                %th= _("Node")
+                %th= _("State")
+              - @resource.running_on.each do |node, state|
+                %tr
+                  %td= content_tag :code, node
+                  %td= content_tag :code, state || "-"
 
-        - if @resource.failed_ops.any?
-          .panel.panel-default
-            .panel-heading#r-failures{role: :tab}
-              %h4.panel-title
-                %a{role: :button, data: {toggle: :collapse, parent: "#r-info"}, href: "#r-failures-body", aria: {expanded: false, controls: "r-failures-body"}}
-                  = _("Failed Operations")
-                  .pull-right
-                    %span.caret
-            #r-failures-body.panel-collapse.collapse{role: :tabpanel, aria: {labelledby: "r-failures"}}
-              .panel-body
-                %table.table
-                  %tr
-                    %th= _("Node")
-                    %th= _("Call")
-                    %th= _("Operation")
-                    %th= _("Status")
-                    %th= _("Reason")
-                  - @resource.failed_ops.each do |op|
-                    %tr
-                      %td= content_tag :code, op[:node]
-                      %td= content_tag :code, op[:call_id]
-                      %td= content_tag :code, op[:op]
-                      %td= content_tag :code, op[:rc_code]
-                      %td= content_tag :code, op[:exit_reason]
+      - if @resource.failed_ops.any?
+        .panel.panel-default
+          .panel-heading#r-failures{role: :tab}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace